content/articles/soft-deletion.md

```diff
@@ -174,14 +174,14 @@ FROM deleted
 RETURNING *;
 ```
 
-This does have a downside compared to `deleted_at` -- the process of selecting columns into `jsonb` isn't easily reversible. While it's possible to do so, it would likely involve building one-off queries and manual intervention. But again, that might be okay -- consider how often you're really going to be trying to undelete data.
+This does have a downside compared to `deleted_at` -- the process of selecting columns into `jsonb` isn't easily reversible. While it's possible to do so, it would likely involve building once-off queries for manual intervention. But again, that might be okay -- consider how often you're really going to be trying to undelete data.
```

Comment:
Would you mind reverting the changes on this line? Believe it or not, both of those were on purpose :)